### ISSUE TITLE
Improvement required for Clio's landing page

### ISSUE THREAD

**Issue by shivanshuraj1333** [eellak/clio#37]
For better UI/UX of Clio, a robust fronted design is needed. I tried to design landing page of Clio, please have a look at it. Suggestions from contributors are most welcome.

**-> Current Landing Page of Clio**

![Screenshot from 2019-03-15 00-11-16](https://user-images.githubusercontent.com/26167974/54384067-6d68f580-46b9-11e9-9faa-22a53eedb34c.png)

**-> My new Design**

![Screenshot from 2019-03-15 00-04-49](https://user-images.githubusercontent.com/26167974/54384135-9093a500-46b9-11e9-9792-11eccf0972f4.png)

![Screenshot from 2019-03-15 00-05-37](https://user-images.githubusercontent.com/26167974/54384143-92f5ff00-46b9-11e9-91b6-24f8901186b8.png)

**The new Design Includes**

1.  A Tour of the website for better user experience
2.  Header and footer
3. A newly Designed Clio's logo (still open for suggestions)
4. A sign in option to register new users
5. Animation for better user experience 
-> This is just an initial design to get feedback for further improvement.
-> I will also try to implement material design and will compare the result.
**Please give feedback, Thank you!**

**Comment by shivanshuraj1333** [comment]
New landing page in motion...
![ezgif com-video-to-gif](https://user-images.githubusercontent.com/26167974/54385873-ac994580-46bd-11e9-8783-a530de5a2306.gif)

**Comment by zvr** [comment]
I don't think the background pictures help; keep in mind that any installation would be used a lot, by a lot of people.

Thinking more about it, you conflate two different functionalities: the actual clio instance where info is kept and information about clio the software. Do these belong on the same site?

**Comment by shivanshuraj1333** [comment]
@zvr I totally agree with your first point that background picture is of no use, this was just an initial design to start with. Now I am trying with material design.
I don't completely understand your message from second statement. I created a frontend which gives a little information about Clio and on scrolling down user can find the Actual Clio instance where information is kept about license, component and products.
Please elaborate your point.

**Comment by zvr** [comment]
The two functionalities, or contents, or web sites are:
1. a clio instance which has data and is used extensively; and
2. a website explaining the software clio.

Think of (1) like a blog of some kind (or a similar web site) powered by wordpress and (2) the site  https://wordpress.org/.  You don't have explanations about the wordpress software in every site created by wordpress.
[feel free to substitute Drupal instead of Wordpress in the example, if you want]

**Comment by shivanshuraj1333** [comment]
@zvr now I understand it well.
So, apart from enhancing UI/UX of clio, I will add a Know More button which will redirect user to a Blog containing information about clio.
And if needed I will also add a FAQ button on the main website itself. I think these things will surely give any user complete info about clio.

**Comment by shivanshuraj1333** [comment]
> @zvr now I understand it well.
> So, apart from enhancing UI/UX of clio, I will add a Know More button which will redirect user to a Blog containing information about clio.
> And if needed I will also add a FAQ button on the main website itself. I think these things will surely give any user complete info about clio.

@zvr is this new design is okay with you? I want to initiate this work.

**Comment by gopuvenkat** [comment]
The documentation of clio (powered by sphinx) is maintained at https://clio.ellak.gr/doc/.

**Comment by shivanshuraj1333** [comment]
Thanks @gopuvenkat.
